src/stores/cache.js

```diff
@@ -32,5 +37,9 @@ export default class Cache {
   serialize() {
     return this.cache;
   }
+
+  setPerma(perma) {
```

Comment:
I feel like this could be named better... At first I leaned towards `setEnabled` but I guess that's not _quite_ right... But maybe something that explains how it affects the cache - i.e. if this is set to `true`, nothing expires.